caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -238,6 +238,15 @@ func TestBuildEnv(t *testing.T) {
 	envExpected = newEnv()
 	envExpected["SCRIPT_NAME"] = "/test/fgci_test.php"
 	testBuildEnv(r, rule, fpath, envExpected)
+
+	// 6. Test SCRIPT_NAME,SCRIPT_FILENAME do not include PATH_INFO
```

Comment:
You could probably number this test `7.` (the one before this one is already `6.` 😛 )

Reply:
Oops! Good catch. Fixed.